install/installer.class.php

```diff
@@ -1301,7 +1301,7 @@ static public function getPolicies() {
             'type'                                => 'bool',
             'type_data'                           => '',
             'unicity'                             => 1,
-            'plugin_flyvemdm_policycategories_id' => 'Security > User interface',
+            'plugin_flyvemdm_policycategories_id' => 'Security > Phone',
```

Comment:
@DIOHz0r 
This policy is not related to phoning. I would rather keep it in "Security > User interface".